.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    name: Build and publish distribution to PyPI
```

Comment:
This workflow will run whenever a tag is pushed, to any branch.

The corresponding travis workflow runs only on tags pushed to master.
If this is necessary we need the following inside the `publish` section:
```
if: github.event.base_ref == 'refs/heads/master'
```

Reply:
LGTM